code/SubsitesVirtualPage.php

```diff
@@ -207,4 +207,13 @@ public function init(){
 		
 		Subsite::$disable_subsite_filter = $origDisableSubsiteFilter;
 	}
+	
+	public function allowedActions ($limitToClass = NULL) {
+		$parent = SiteTree::get()->byID($this->CopyContentFromID);
+		$model = $parent ? ModelAsController::controller_for($parent) : null;
+		if ($model && get_class($model) != get_class($this)) {
```

Comment:
If the classes match, then it must be another virtual page, yes?

If so, then shouldn't it take the actions from that virtual page (rather than just skipping as this code is doing).